src/client/app/styles.css

```diff
@@ -0,0 +1,38 @@
+body {
```

Comment:
May want to consider the use of CSS Modules, https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822  ...  takes some webpack setup though